port/win/port_win.cc

```diff
@@ -142,7 +142,7 @@ DIR* opendir(const char* name) {
     return nullptr;
   }
 
-  strncpy_s(dir->entry_.d_name, dir->data_.name, strlen(dir->data_.name));
+  strcpy_s(dir->entry_.d_name, sizeof(dir->entry_.d_name), dir->data_.name);
```

Comment:
Ah, you're correct. I think the point is that you can omit the second parameter in your call to take advantage of the template magic explained in https://msdn.microsoft.com/en-us/library/ms175759.aspx (for which I added support in the above commit).

I don't know why this was using `strncpy_s` before.